### PR TITLE
Feature/smka 98 audioguide bypass

### PR DESCRIPTION
**Pull request by rasben**

### CODE REVIEW

File: sites/all/modules/smk_audio/smk_audio.ds.inc [hunk 1]
```diff
@@ -328,8 +328,8 @@ function smk_audio_ds_layout_settings_info() {
       'ds_content' => array(
         0 => 'group_mediaplayer',
         1 => 'group_info_panel',
-        2 => 'group_content',
-        3 => 'sound_mediaplayer_metadata',
+        2 => 'sound_mediaplayer_metadata',
+        3 => 'group_content',
```
Comment (kallehauge) [inline sites/all/modules/smk_audio/smk_audio.ds.inc]: Is this on purpose? (order change)

File: sites/all/modules/smk_audio/smk_audio.ds.inc [hunk 2]
```diff
@@ -347,8 +347,8 @@ function smk_audio_ds_layout_settings_info() {
     'fields' => array(
       'group_mediaplayer' => 'ds_content',
       'group_info_panel' => 'ds_content',
-      'group_content' => 'ds_content',
       'sound_mediaplayer_metadata' => 'ds_content',
+      'group_content' => 'ds_content',
```
Comment (kallehauge) [inline sites/all/modules/smk_audio/smk_audio.ds.inc:350]: Is this on purpose? (order change)

File: sites/all/modules/smk_audio/smk_audio.module [hunk 3]
```diff
@@ -465,3 +465,4 @@ function smk_audio_tokens($type, $tokens, array $data = array(), array $options
   return $replacements;
 }
 
+
```
Comment (kallehauge) [inline sites/all/modules/smk_audio/smk_audio.module]: This seems wrong :smile:

**Comment by kallehauge** [comment]
In general it looks good 👍 I have a few questions and found a single double new line to end a file.

File: sites/all/modules/smk_audio/smk_audio.ds.inc [hunk 4]
```diff
@@ -328,8 +328,8 @@ function smk_audio_ds_layout_settings_info() {
       'ds_content' => array(
         0 => 'group_mediaplayer',
         1 => 'group_info_panel',
-        2 => 'sound_mediaplayer_metadata',
-        3 => 'group_content',
+        2 => 'group_content',
+        3 => 'sound_mediaplayer_metadata',
         4 => 'field_audio_file',
```
Comment (rasben) [inline sites/all/modules/smk_audio/smk_audio.ds.inc:333]: ^
I have re-ordered the features, back to what they were :) I dont think it made any difference, but now we're sure.

File: sites/all/modules/smk_audio/smk_audio.ds.inc [hunk 5]
```diff
@@ -347,8 +347,8 @@ function smk_audio_ds_layout_settings_info() {
     'fields' => array(
       'group_mediaplayer' => 'ds_content',
       'group_info_panel' => 'ds_content',
-      'sound_mediaplayer_metadata' => 'ds_content',
       'group_content' => 'ds_content',
+      'sound_mediaplayer_metadata' => 'ds_content',
       'field_audio_file' => 'ds_content',
```
Comment (rasben) [inline sites/all/modules/smk_audio/smk_audio.ds.inc:352]: ^
I have re-ordered the features, back to what they were :) I dont think it made any difference, but now we're sure.

**Comment by kallehauge** [comment]
Review OK :+1:

**Pull request merged**